### ISSUE TITLE
Cherrypick PRs Shouldn't Require Approval

### ISSUE THREAD

**Issue by grodrigues3** [kubernetes/test-infra#1828]
Cherrypicks that have already been merged into master should not require approval

**Comment by grodrigues3** [comment]
@Sevii may require gaining some familiarity with the Approvers process, but also something worth getting started on

**Comment by spxtr** [comment]
When we cut a release branch, we can remove all OWNERS files except for the top-level file, and set the release owner as the approver within it. Then our OWNERS implementation doesn't need to change :)

**Comment by ixdy** [comment]
@spxtr that might make cherrypicks and fast-forwards somewhat trickier.

**Comment by spxtr** [comment]
How so?

**Comment by ixdy** [comment]
I guess my main concern is this situation:
1. We branch `release-x.y` from `master` for `x.y.0-beta.0`
2. We remove all OWNERS files in the repo, and replace the top-level OWNERS with the release owner
3. We continue adding changes to master, including some that affect OWNERS files
4. We try to fast-foward `release-x.y` from `master`. Now we're probably going to run into conflicts, because there are changes to deleted files, right?

**Comment by spxtr** [comment]
Ideally, changes to OWNERS files won't be cherry-picked, since they will be in their own PRs. You're right though, that probably will be annoying.

**Comment by ixdy** [comment]
Yeah, I'm less worried about cherry-picks and more worried about fast-fowards early in the release cycle (during code freeze), which I guess isn't what this issue is really talking about.

As long as we don't rewrite the OWNERS files until we exit code freeze (i.e. when we cut the RC), this should be OK.

**Comment by fejta** [comment]
Should cherrypicked PRs should get approval from the owner of the release?

**Comment by stevekuznetsov** [comment]
Realistically if `kubernetes` org can move to using the cherry-picker robot it's easier to implement this by looking at the author of the PRs

**Comment by liggitt** [comment]
> Cherrypicks that have already been merged into master should not require approval

I disagree with this. Approval for a change going into master does not automatically mean it is appropriate for a release branch.

**Comment by kargakis** [comment]
> I disagree with this. Approval for a change going into master does not automatically mean it is appropriate for a release branch.

+1
